docling/utils/ocr_utils.py

```
@@ -3,7 +3,7 @@ def map_tesseract_script(script: str) -> str:
     if script == "Katakana" or script == "Hiragana":
         script = "Japanese"
     elif script == "Han":
-        script = "HanS"
+        script = "Japanese"
```

Ruff (0.11.9)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)